exercises/practice/quadratic-roots/quadratic_roots_test.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from quadratic_roots import roots
+import z3
+
+def toFloat(x: z3.RatNumRef) -> float:
+    """Converts a Rational Number from z3 to a float that base python can use."""
+    x = x.as_fraction()
+    return float(x.numerator) / float(x.denominator)
+
+def check(ut, t1: tuple, t2: tuple, places = 7):
+    """Checks if two tuples contain the same nearly equivalent elements, regardless of order."""
+    if(t1 is None and t2 is None):
+        ut.assertTrue(True)
+    elif(t2 is None or t2 is None):
+        # technically an xor
+        ut.assertTrue(False)
+    else:
+        t1 = tuple(toFloat(x) for x in t1)
+        ut.assertTrue((round(t1[0], places) == round(t2[0], places) and round(t1[1], places) == round(t2[1], places)) or
```

Comment:
Instead of rounding each number here, the z3 function set_option(precision=7) could be used, similar to how it is used in the unittest.TestCase child class 'setUp' method here: https://github.com/CSwift8/z3/blob/equilateral-triangle/exercises/practice/equilateral_triangle/equilateral_triangle_test.py#L15

Reply:
Thanks!